examples/GroceryExample/MyLabelSet.cs

```diff
@@ -0,0 +1,39 @@
+// <copyright file="MyLabelSet.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System.Collections.Generic;
+using OpenTelemetry.Metrics;
+
+#pragma warning disable CS0618
+
+namespace GroceryExample
+{
+    public class MyLabelSet : LabelSet
```

Comment:
trying to understand why did you create `MyLabelSet`?.... Help me understand.

Reply:
LabelSet is an abstract class in API.  And I don't want to take a dependency on SDK's LabelSetSdk.

Reply:
I see. You shouldn't need to create own LabelSet.. Meter exposes methods to create LabelSet. https://github.com/open-telemetry/opentelemetry-dotnet/blob/ab278ceccbf1e11a3829cad85a85d8a81ebb8539/src/OpenTelemetry.Api/Metrics/Meter.cs#L83

(you can ignore this comment as this code will anyway change a lot. but just sharing what was the original attempt)

Reply:
I see you have a Meter.GetLabelSet() virtual method.  I'll try to refactor using that.